plutus-playground/plutus-playground-client/src/Editor.purs

```diff
@@ -81,6 +83,9 @@ editorPane state =
                         , text "Please try again or contact support for assistance."
                         ]
                     _ -> empty
+      warningList = case state.compilationResult of
+                     (Success (Right result)) -> listGroup_ (listGroupItem_ <<< pure <<< compilationWarningPane <$> (view (_CompilationResult <<< _warnings) result))
```

Comment:
I would be tempted to spin this out into a `compilationWarningsPane :: Array Warning -> HTML p Query`. As it stands this statement is on the very edge of readability (IMO).